site2/website/pages/en/events.js

```diff
@@ -57,13 +60,32 @@ class Events extends React.Component {
             <MarkdownBlock>
               - [Pulsar Developer Community biweekly meetup](https://github.com/streamnative/pulsar-community-loc-cn/)
             </MarkdownBlock>
+            <h2><translate>Groups</translate></h2>
+            <MarkdownBlock>
+              - [NorCal Apache Pulsar Neighborhood Meetup Group](https://www.meetup.com/nor-cal-apache-pulsar-meetup-group/)
+            </MarkdownBlock>
+            <MarkdownBlock>
+              - [Netherlands Apache Pulsar Meetup Group](https://www.meetup.com/netherlands-apache-pulsar-meetup/)
+            </MarkdownBlock>
+            <MarkdownBlock>
+              - [SoCal Apache Pulsar Neighborhood Meetup Group](https://www.meetup.com/socal-apache-pulsar-meetup-group/)
+            </MarkdownBlock>
+            <MarkdownBlock>
+              - [New York City Apache Pulsar Meetup](https://www.meetup.com/new-york-city-apache-pulsar-meetup/)
+            </MarkdownBlock>
+            <MarkdownBlock>
+              - [Beijing Apache Pulsar® Meetup by StreamNative](https://www.meetup.com/beijing-apache-pulsar-meetup-by-streamnative/)
+            </MarkdownBlock>
```

Comment:
There are many Pulsar meetups / webinars / online courses worldwide, should we add them here?

Reply:
I agree with @eolivelli lets do it one step at a time, but think about what makes sense here.
I would like to have a place where someone could see all of the events that are going on, both large and small.  I guess the calendar above would be a good place for that.
For the Groups, I think that we just keep adding until we need to organize it better (great problem to have). 
But let's take it one step at a time.  I am figuring that someone is going to need to update this page about every 2-3 weeks.  I am more than happy to keep adding in what I learn about, but I also want others to add in content.